dockerfile/directx/win10.dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM mcr.microsoft.com/windows:2004
+
+
+# Install Python and additional packages
+# Download Python
+ADD https://www.python.org/ftp/python/3.9.7/python-3.9.7-amd64.exe python-installer.exe
+# Install Python
+RUN python-installer.exe /quiet InstallAllUsers=1 PrependPath=1 && DEL python-installer.exe
+# Verify Python Was Successfully Installed
+RUN python --version
+RUN python -m ensurepip --upgrade
```

Comment:
merge into one stage?

Reply:
need a new shell to check 'python --version', so not merge line#8 and line#10